infra/views.py

```diff
@@ -0,0 +1,17 @@
+'''Provide API views for infra module.'''
+from rest_framework import viewsets
+
+import infra.models
+import infra.serializers
+
+
+class OperationLogViewSet(viewsets.ModelViewSet):
```

Comment:
https://github.com/DLUT-SIE/TMSFTT-BE/blob/0496155dfdffe018d5a1a19efdb6649098ecf6d4/infra/views.py#L8-L11
Delete the whole section and test code related.